hello.py

```
@@ -31,4 +32,4 @@ def say_hello():
 
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run(app, host="0.0.0.0", port=8003)
\ No newline at end of file
+    uvicorn.run(app, host="0.0.0.0", port=HELLO_SERVICE_PORT)
```

Ruff (0.11.9)

`HELLO_SERVICE_PORT` may be undefined, or defined from star imports

(F405)